Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1072,10 +1072,43 @@ std::variant<bool, CLuaMultiReturn<float, float, float, float>> CLuaPedDefs::Get
     return std::make_tuple(x, y, z, w);
 }
 
-bool CLuaPedDefs::SetElementBoneMatrix(lua_State* const luaVM, CClientPed* entity, std::uint32_t boneId, CMatrix boneMatrix)
```

Comment:
Just implement that "CMatrix" can be read from 4x4 lua table and don't touch this file at all